### PR TITLE
Add basic dynamic typing to ASTNode

### PR DESCRIPTION
**Pull request by pniedzwiedzinski**
Change ASTNode.value to string (instead of int) so we can store different values

### CODE REVIEW

**Comment by mhnowak** [comment]
As I said before imo it'd be better to create an abstract class `Resolver` right now with 1 method `execute`. It starts to get really messy here on the main file. Obviously you'd need to create something like `Computation` or something else to calculate on values.

File: src/pdc.h [hunk 1]
```diff
@@ -1,10 +1,10 @@
+#include <string>
+
 #define ADD 	'+'
 #define SUBSTRACT	'-'
 #define MULTIPLY	'*'
 #define DIVIDE	'/'
 
-//TODO: Dynamic typing, for now every value is int
-#define PseudoValue int // To enable dynamic typing we need to use one type for every value
+#define PseudoValue std::string // To enable dynamic typing we need to use one type for every value
```
Comment (mhnowak) [inline src/pdc.h:8]: What about `* char` instead of string

Comment (pniedzwiedzinski) [reply src/pdc.h:8]: With string it's easier to convert to int :laughing:

Comment (pniedzwiedzinski) [reply src/pdc.h:8]: And in reverse

**Comment by pniedzwiedzinski** [comment]
Superseded by #5

**Pull request closed without merging**